Index: .gitignore
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>\"*.class\" \r\n\"production/\" \r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.gitignore b/.gitignore
--- a/.gitignore	(revision 1e5defda10706c7dc57652990b862459a23d8195)
+++ b/.gitignore	(date 1756642723009)
@@ -1,2 +1,10 @@
 "*.class" 
 "production/" 
+.gradle/
+build/
+.idea/workspace.xml
+.idea/tasks.xml
+.idea/usage.statistics.xml
+.idea/shelf/
+*.class
+
